_episodes/14-writing-functions.md

```diff
@@ -77,17 +77,14 @@ print_date(1871, 3, 19)
 ~~~
 {: .output}
 
-*   Via [Twitter](https://twitter.com/minisciencegirl/status/693486088963272705):
```

Comment:
If I am making a quiche, the ingredients for the filling are certainly eggs and milk, but also some sort of green vegetable, some sort of cheese, and perhaps some sort of meat.  While the eggs and milk are strictly specified, there is flexibility in other ingredients.  While the analogy of recipes to function definitions is imperfect, I think many people learn by extening known patterns to new concepts, (eg, "Oh, this [new idea] is like [another old idea] that I already understand.  Shayer and Adey call this "bridging" or "transfer" - [see this pdf](http://modeling.asu.edu/modeling/AdeyP-CognitiveAccel2011.pdf) )

Again, I would prefer to keep this as an anecdote at least.  I've noticed the writing in this section is quite terse.  Perhaps if it was presented explicity as an anecdote with explaination...